monitor/test_result.py

```diff
@@ -138,17 +138,20 @@ def __init__(
         self.table_full_name = '.'.join(table_full_name_parts).lower()
         self.detected_at = None
         if detected_at:
```

Comment:
I know it has been this way even before but maybe we should also init self.detected_at_utc with None before this if statement?